fe1-web/store/reducers/__tests__/SocialReducer.test.ts

```diff
@@ -145,6 +184,21 @@ test('reducer should add the newest chirp on top', () => {
     .toEqual(filledState4);
 });
 
+test('reducer should mark chirp 1 as deleted', () => {
+  expect(socialReduce(filledState4, deleteChirp(mockLaoId, chirp1Deleted)))
+    .toEqual(filledState5);
+});
+
+test('delete a non-stored chirp should do nothing', () => {
+  expect(socialReduce(filledState4, deleteChirp(mockLaoId, chirp0)))
+    .toEqual(filledState4);
+});
+
+test('reducer should not re-add a chirp if it has already been deleted', () => {
+  expect(socialReduce(filledState5, addChirp(mockLaoId, chirp1Deleted)))
+    .toEqual(filledState5);
+});
```

Comment:
```suggestion
test('reducer should not re-add a chirp if it has already been deleted', () => {
  var stateDeleted = socialReduce(filledState3, deleteChirp(mockLaoId, chirp4));
  expect(socialReduce(stateDeleted, addChirp(mockLaoId, chirp4)))
    .toEqual(filledState4);
});
```

would the following work ? I guess not, but it's entirely possible that in the future we get messages out of order (e.g., if we're connected to multiple servers)

Reply:
With the current implementation, this test passes (first delete a non-stored chirp, then add it).
So, if I understand correctly, when we receive out-of-order messages (it should be: add then delete, but we get: delete then add), the expected state should mark the chirp as deleted (toEqual(filledState5))?